devtools/pip-tools/pr_tool/main.py

```diff
@@ -215,7 +215,9 @@ def __init__(self, commits):
     def check_commit(self, commit):
         headline, *messageBody = commit.message.split("\n")
         issues = []
-        if messageBody[0] == "":
+        if len(messageBody) == 0:
+            pass
```

Comment:
I think we should add an issue here since the headline will only not contain a terminating "\n" in the case something went wrong.